internal/service/advancedclustertpf/resource_compatibility_reuse.go

```diff
@@ -11,6 +12,23 @@ import (
 	"go.mongodb.org/atlas-sdk/v20241113002/admin"
 )
 
+func IsMajorVersionHigherGreaterOrEqual(input *string, version float64) bool {
+	if input == nil || *input == "" {
+		return true
+	}
```

Comment:
```suggestion
	if !conversion.IsStringPresent(input) {
		return true
	}
```

Reply:
Changed in 290b393e5fff8730bcd18f0f501c2d74d3a2469c